technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -434,6 +418,24 @@
       </choose>
     </group>
   </macro>
+  <macro name="volume">
+    <group suffix=", ">
+      <number suffix=" " variable="number-of-volumes"/>
+      <text term="number-of-volumes" form="short"/>
+    </group>
+    <group>
+      <choose>
+        <if match="any" is-numeric="volume">
+          <text term="volume" form="short" suffix=" "/>
+          <number variable="volume"/>
+        </if>
+        <else>
+          <text term="volume"/>
+          <text variable="volume"/>
```

Comment:
@jmiba, I think you need a delimiter between the term and variable here, like a space.